From ae70d66bf4d4781da7a67f4f7095c1f34c8c8a42 Mon Sep 17 00:00:00 2001
From: Nathan VanBenschoten <nvanbenschoten@gmail.com>
Date: Mon, 17 Oct 2016 15:44:51 -0400
Subject: [PATCH] CMakeLists.txt: only call roslint if testing is enabled

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dd7ed3f..63172f6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,4 +15,6 @@ install(PROGRAMS
 
 # Check package for pep8 style, add a test to fail on violations.
 roslint_python()
-roslint_add_test()
+if(CATKIN_ENABLE_TESTING)
+   roslint_add_test()
+endif()
-- 
2.7.4

